requirements.txt

```diff
@@ -0,0 +1,4 @@
+Fabric==1.10.1
+PyYAML==3.11
+boto==2.36.0
```

Comment:
We do in the case of a PIP package, but when you're developing you need to be inside a virtual env and install them yourself.

If you think it's cleaner I can rename the file requirements-test.txt or something.